features/minion_ssh_push.feature

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2016 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: register a salt-minion via SSH push
+
+  Scenario: bootstrap a sles minion
+     Given I am authorized
+     When I follow "Salt"
+     Then I should see a "Bootstrapping" text
+     And I follow "Bootstrapping"
+     Then I should see a "Bootstrap Minions" text
+     # minion = sles, rh_minion = redhat
+     And  I enter the hostname of "sle-minion" as hostname
+     And I enter "22" as "port"
+     And I enter "root" as "user"
+     And I enter "linux" as "password"
+     And I click on "Bootstrap it"
+     Then i should see a "Successfully bootstrapped host! Your system should appear in System Overview shortly." text
```

Comment:
HERE maybe we need to wait until the minion is bootstrapped. We should check that same process still exist on the command line, then wer are done, to wait. ( don't know which process could be.)